docs/source/conf.py

```diff
@@ -73,3 +74,6 @@
 html_css_files = [
     "css/max_width.css",
 ]
+
+# Show the defaults for the keyword args.
+typehints_defaults = "comma"
```

Comment:
The lack of a newline at the end is throwing a formatting error with black. In any case, I was thinking this would fit better after line 60 instead